src/AuthModule/Components/Login/Login.tsx

```diff
@@ -1,7 +1,107 @@
-import React from 'react'
-
+import React from "react";
+import TextField from "@mui/material/TextField";
+import Container from "@mui/material/Container";
+import Typography from "@mui/material/Typography";
+import { useForm } from "react-hook-form";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import loginImage from "../../../assets/login.png";
+import { Avatar } from "@mui/material";
+import LockPersonOutlinedIcon from "@mui/icons-material/LockPersonOutlined";
+import Button from "@mui/material/Button";
+import Link from "@mui/material/Link";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import { toast } from "react-toastify";
 export default function Login() {
+  let navigate=useNavigate();
+  const paperStyle = {
+    padding: 20,
+    height: "60vh",
+    width: 280,
+    margin: "20px auto",
+  };
+  const avaterStyle = { color: "#3252DF" };
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm();
+
+  const onSubmit=async (data:object)=>{
+    try{ 
+      let response=await axios.post(`${baseUrl}/admin/users/login`,data);
+      toast.success("Welcome!");
+    navigate('/Admin')
+  }
+  catch(error){
+    toast.error(error?.response?.data?.message || "There's a mistake.");
+
+  }
+   
+
+  }
+ 
+
   return (
-    <div>Login</div>
-  )
+    <>
+      <Grid>
+        <Paper style={paperStyle} elevation={10}>
+          <Grid textAlign={"center"}>
+            <LockPersonOutlinedIcon style={avaterStyle} />
+            <h2 className="mt-2">Sign in</h2>
+          </Grid>
+          <Grid>
```

Comment:
We are not using anything related to the  MUI Grid layout here , also the form is only one dimensional, so we can use the stack instead for giving spacing between elements.

We can also add the component prop and make it `component="form"` and then give it the  `onSubmit={handleSubmit(onSubmit)}` and remove the form element below.